_includes/_author-bio.html

```diff
@@ -32,3 +32,4 @@ <h3 itemprop="name">{{ author.name }}</h3>
   {% if author.weibo %}<a href="http://www.weibo.com/{{ author.weibo }}" class="author-social" target="_blank"><i class="fa fa-fw fa-weibo"></i> Weibo</a>{% endif %}
   {% if author.flickr %}<a href="http://www.flickr.com/{{ author.flickr }}" class="author-social" target="_blank"><i class="fa fa-fw fa-flickr"></i> Flickr</a>{% endif %}
 </div>
+<a href="{{ site.url/feed.xml }}" class="author-social" target="_blank"><i class="fa fa-fw fa-rss"></i> RSS </a>
```

Comment:
@srijanshetty i dont think your code works!

It should be: {{ site.url }}/feed.xml instead of {{ site.url/feed.xml }}.